playroom/components/index.tsx

```diff
@@ -90,6 +115,11 @@ const themesMap: {
         themeConfig: O2,
         icon: <O2Logo size={24} />,
     },
+    'O2-new': {
+        text: 'O2 new',
+        themeConfig: O2_New,
+        icon: <O2NewLogo />,
```

Comment:
no size?

Reply:
Nope, this icon is a custom asset to show "New" badge like in Vivo New

![image](https://github.com/Telefonica/mistica-web/assets/6722153/fe041feb-f7f5-468f-8155-9a92393a2e48)

When we remove standard "o2" skin from previewTools we will use it from Logo component (without the badge new)